app/src/shared/components/Subscription/index.jsx

```diff
@@ -60,28 +76,58 @@ class Subscription extends Component {
         // unsubscribe if switching to active
         if (this.props.isActive && !this.isSubscribed) {
             this.autosubscribe()
-        } else if (!this.props.isActive && !this.isSubscribed) {
+        } else if (!this.props.isActive && this.isSubscribed) {
```

Comment:
👍 I thought I already fixed this but alas I mustn't have cherry picked the commit, or maybe it got lost in the refactoring https://github.com/streamr-dev/streamr-platform/pull/752/files would be good if this was covered by a test